community-operators/postgresql/4.1.0/postgresoperator.v4.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,549 @@
+      - description: The current status of the application.
+        displayName: Status
+        path: phase
+        x-descriptors:
+        - urn:alm:descriptor:io.kubernetes.phase
+      - description: Explanation for the current status of the application.
+        displayName: Status Details
+        path: reason
+        x-descriptors:
+        - urn:alm:descriptor:io.kubernetes.phase:reason
+      version: v1
+    - description: Represents a Postgres replica cluster member
+      displayName: Postgres Replica Cluster Member
+      kind: Pgreplica
+      name: pgreplicas.crunchydata.com
+      resources:
+      - kind: Deployment
+        name: ''
+        version: v1
+      - kind: ReplicaSet
+        name: ''
+        version: v1
+      - kind: Pod
+        name: ''
+        version: v1
+      - kind: Service
+        name: ''
+        version: v1
+      specDescriptors:
+      - description: The desired number of member Pods for the deployment.
+        displayName: Size
+        path: size
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:podCount
+      statusDescriptors:
+      - description: The current status of the application.
+        displayName: Status
+        path: phase
+        x-descriptors:
+        - urn:alm:descriptor:io.kubernetes.phase
+      - description: Explanation for the current status of the application.
+        displayName: Status Details
+        path: reason
+        x-descriptors:
+        - urn:alm:descriptor:io.kubernetes.phase:reason
+      version: v1
+    - description: Represents a Postgres sql policy
+      displayName: Postgres SQL Policy
+      kind: Pgpolicy
+      name: pgpolicies.crunchydata.com
+      resources:
+      - kind: Pgpolicy
+        name: ''
+        version: v1
+      specDescriptors:
+      - description: The pgpolicy name.
+        displayName: Name
+        path: name
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:name
+      - description: The pgpolicy sql.
+        displayName: SQL
+        path: sql
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:sql
+      statusDescriptors:
+      - description: The current status of the application.
+        displayName: Status
+        path: phase
+        x-descriptors:
+        - urn:alm:descriptor:io.kubernetes.phase
+      - description: Explanation for the current status of the application.
+        displayName: Status Details
+        path: reason
+        x-descriptors:
+        - urn:alm:descriptor:io.kubernetes.phase:reason
+      version: v1
+    - description: Represents a Postgres workflow task
+      displayName: Postgres workflow task
+      kind: Pgtask
+      name: pgtasks.crunchydata.com
+      resources:
+      - kind: Pgtask
+        name: ''
+        version: v1
+      specDescriptors:
+      - description: The pgtask name.
+        displayName: Name
+        path: name
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:name
```

Comment:
Same here, if this path ("name") is expecting a `type: string`, recommend using `urn:alm:descriptor:com.tectonic.ui:text` (see details in "Text Input") as the `x-descriptors`.